ui-components/src/actions/ButtonGroup.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from "react"
+import { BADGES } from "../../.storybook/constants"
+import { AppearanceStyleType } from "../.."
+import Button from "./Button"
+import ButtonGroupDocumentation from "./ButtonGroup.docs.mdx"
+
+import { ButtonGroup, ButtonGroupSpacing } from "./ButtonGroup"
+
+export default {
+  title: "Actions/Button Group 🚩",
+  id: "actions-button-group",
+  parameters: {
+    docs: {
+      page: ButtonGroupDocumentation,
+    },
+    badges: [BADGES.GEN2],
+  },
+}
+
+export const twoColumns = () => (
+  <ButtonGroup
+    columns={[
+      <Button>Previous</Button>,
+      <Button styleType={AppearanceStyleType.primary}>Next</Button>,
+    ]}
+    spacing={ButtonGroupSpacing.between}
+  />
+)
+
+export const oneColumnLeft = () => (
+  <ButtonGroup
+    columns={[
+      <>
+        <Button>Previous</Button>
+        <Button styleType={AppearanceStyleType.primary}>Next</Button>
+      </>,
+    ]}
+  />
+)
+
+export const oneColumnRight = () => (
+  <ButtonGroup
+    columns={[
+      <></>,
+      <>
+        <Button>Previous</Button> <Button styleType={AppearanceStyleType.primary}>Next</Button>
+      </>,
+    ]}
+  />
+)
+
+export const oneColumnRightReversed = () => (
```

Comment:
This title confused me when first looking at the storybook. It might just be me, but I think oneColumnReversed is better name. That way is distinguishable from the one above